stats/stats.go

```diff
@@ -53,22 +52,35 @@ func Import(store Store) error {
 	return nil
 }
 
-// ImportRegularly runs Import to import aggregated stats from a remote server at regular intervals.
-func ImportRegularly(store Store, interval time.Duration) {
+// Update imports aggregated scans and update our cache table of local scans.
+// Log any errors.
+func Update(store Store) {
+	err := Import(store)
+	if err != nil {
+		log.Println(err)
+		raven.CaptureError(err, nil)
+	}
+	// Cache stats for the previous day at midnight. This ensures that we capture
+	// full days and maintain regularly intervals.
+	_, err = store.PutLocalStats(time.Now().UTC().Truncate(24 * time.Hour))
+	if err != nil {
+		log.Println(err)
+		raven.CaptureError(err, nil)
+	}
```

Comment:
Let's capture where the error is coming from in case we get a sentry report from here, then there's more context for debugging. Prepending something like "[Regular Stats Importer]" here, or a better name.